projects/plugins/crm/src/automation/commons/conditions/contacts/class-contact-status-changed.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Jetpack CRM Automation Contact_Status_Changed condition.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Conditions;
+
+use Automattic\Jetpack\CRM\Automation\Automation_Exception;
+use Automattic\Jetpack\CRM\Automation\Automation_Logger;
+use Automattic\Jetpack\CRM\Automation\Base_Condition;
+
+/**
+ * Contact_Status_Changed condition class.
+ *
+ * @since $$next-version$$
+ */
+class Contact_Status_Changed extends Base_Condition {
+
+	/**
+	 * The Automation logger.
+	 *
+	 * @var Automation_Logger $logger The Automation logger.
+	 */
+	private $logger;
+
+	/**
+	 * All valid operators for this condition.
+	 *
+	 * @var string[] $valid_operators Valid operators.
+	 */
+	private $valid_operators = array(
+		'is',
+		'is_not',
+	);
+
+	/**
+	 * All valid attributes for this condition.
+	 *
+	 * @var string[] $valid_operators Valid attributes.
+	 */
+	private $valid_attributes = array(
+		'field',
+		'operator',
+		'value',
+	);
+
+	/**
+	 * Contact_Status_Changed constructor.
+	 *
+	 * @param array $step_data The step data for the condition.
+	 */
+	public function __construct( array $step_data ) {
+		parent::__construct( $step_data );
+
+		$this->logger = Automation_Logger::instance();
+	}
+
+	/**
+	 * Executes the condition. If the condition is met, the value stored in the
```

Comment:
For me is super fine as it is... but do we need to add the summary title `Executes the condition.` add a blank line and then the description `If the condition is met,...`?

Reply:
A summary is optional when writing PHPDocs from my understanding.

Reply:
👍  For me, too... asking for it just in case